Makefile

```diff
@@ -375,7 +380,7 @@ ARTIFACTS ?= ${ROOT_DIR}/_artifacts
 KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
 
 .PHONY: test
-test: $(SETUP_ENVTEST) ## Run unit and integration tests
+test: $(SETUP_ENVTEST) download-cluster-api-crd ## Run unit and integration tests
```

Comment:
I'm not sure about adding this as required dependency for test since ideally you could run unit tests offline.

Reply:
Yes, good point. Will fix it.

Reply:
Fixed